paddle/fluid/inference/api/analysis_config.cc

```diff
@@ -435,6 +438,41 @@ void AnalysisConfig::EnableMkldnnBfloat16() {
   Update();
 }
 
+void AnalysisConfig::EnableMkldnnInt8(std::unordered_set<std::string> op_list) {
+#ifdef PADDLE_WITH_MKLDNN
+  use_mkldnn_int8_ = true;
+  use_fc_padding_ = false;
+  if (!op_list.empty()) {
+    for (auto &type : op_list) {
+      if (!quantize_enabled_op_types_.count(type)) {
+        LOG(ERROR) << "There are unsupported operators in the configured "
+                      "quantization operator list. The unsupported operator "
+                      "is: "
+                   << type;
+        use_mkldnn_int8_ = false;
+        break;
+      }
+    }
+    if (use_mkldnn_int8_) {
+      quantize_enabled_op_types_.clear();
+      quantize_enabled_op_types_.insert(op_list.begin(), op_list.end());
+    }
+  }
+  if (quantize_enabled_op_types_.count("fc")) {
+    auto idx = pass_builder_->GetPassIndex("repeated_fc_relu_fuse_pass");
+    if (static_cast<int>(idx) != -1) {
+      pass_builder_->InsertPass(idx, "fc_mkldnn_pass");
+      pass_builder_->InsertPass(idx + 1, "fc_act_mkldnn_fuse_pass");
```

Comment:
I think at this moment, pass_builder is still empty.  
I think these `InsertPass execution` should be put inside `void CpuPassStrategy::EnableMkldnnInt8()`